ophyd/utils/epics_pvs.py

```diff
@@ -319,7 +319,7 @@ def data_type(val):
         if isinstance(val, py_types):
             return json_type
 
-    raise ValueError(
+    raise NonPVValue(
```

Comment:
Might want a second new sub-class of ValueError here.

Reply:
Sorry for the late comment. I am new to GitHub...

I am a bit hesitant on creating exception class names. My impression is that a one which is wisely chosen facilitates usage.

Reply:
Subclassing built-in Exceptions for a package is good practice, as we can make - and catch - very specific errors.

`ValueError` could be raised just about anywhere, but `NonPVValue` could only come from here.